reco_utils/evaluation/python_evaluation.py

```diff
@@ -651,14 +648,16 @@ def get_top_k_items(
         k (int): number of items for each user
 
     Returns:
-        pd.DataFrame: DataFrame of top k items for each user
+        pd.DataFrame: DataFrame of top k items for each user, sorted by `col_user` and `"rank"`
```

Comment:
i would remove the double quotes from rank to match just the backticks like col_user

Reply:
also, in the returns section =)

Reply:
good catch!